tests/e2e/scale/noobaa/test_hsbench.py

```diff
@@ -40,6 +43,7 @@ def teardown():
 
 
 @scale
+@ignore_leftovers
```

Comment:
why you are using this ?

Reply:
@tiffanyn108 any specific reason for ignore leftovers?

Reply:
Fixed and removed ignore_leftover tag.